department-of-veterans-affairs/va.gov-cms
# VACMS-3910: Update formatter to use unmanaged files.

## Description
See #3910 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/106175891-a0f58180-6164-11eb-8cde-dc879dfaf845.png)

## QA steps
- As an administrator, go to `/node/15068/edit` and visually verify that each panel has an svg, and alt text.


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
